gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -29,4 +29,8 @@ export class NotificationSettingsService {
   getNotificationSettings(apiId: string): Observable<NotificationSettings[]> {
     return this.http.get<NotificationSettings[]>(`${this.constants.env.baseURL}/apis/${apiId}/notificationsettings`);
   }
+
+  delete(apiId: string, id: string): Observable<NotificationSettings[]> {
```

Comment:
what do you think about adding a test on this service for this delete ?

Reply:
Done